client/verta/tests/conftest.py

```diff
@@ -572,8 +577,19 @@ def workspace3(client_sys_admin, created_entities):
 ])
 
 
-def create_workspace(client, created_entities, roles):
-    workspace = client._create_workspace(client._conn._get_organization_id(), generate_default_name(), roles)
+@pytest.fixture
+def workspace4(client_sys_admin, created_entities, namespace):
+    return create_workspace(client_sys_admin, created_entities, [
+    RoleV2_pb2.RoleResourceActions(resource_type=RoleV2_pb2.ResourceTypeV2.ENDPOINT,
+                                   allowed_actions=[RoleV2_pb2.ActionTypeV2.READ]),
+    RoleV2_pb2.RoleResourceActions(resource_type=RoleV2_pb2.ResourceTypeV2.REGISTERED_MODEL,
+                                   allowed_actions=[RoleV2_pb2.ActionTypeV2.READ,
+                                                    RoleV2_pb2.ActionTypeV2.UPDATE]),
+], namespace=namespace)
+
+
+def create_workspace(client, created_entities, roles, namespace=""):
```

Comment:
not worried about this being empty most of the time because protos' string default value is the empty string